src/puya/awst/nodes.py

```diff
@@ -228,22 +284,33 @@ def accept(self, visitor: StatementVisitor[T]) -> T:
 
 @attrs.frozen
 class LoopExit(Statement):
-    """break out of the current innermost loop"""
+    """
+    Goto the statement immediately following the current innermost loop construct
```

Comment:
Maybe add something about generally being equivalent to a `break` statement in C-family languages.